### ISSUE TITLE
changed main shortcut, now can't launch emote at all and can't revert

### ISSUE THREAD

**Issue by stephan-dev** [tom-james-watson/Emote#87]
- ubuntu 18.04 (Gnome) (X11) (soon EoL, not yet!)  

From the GUI, I changed the main shortcut (aka the "emoji picker" shortcut). This "worked", as in the new shortcut was displayed, I closed, done.  
But then I totally lost all access to emote. I searched the web and realized that normally, I should have an emote icon somewhere (not specified), which I guess is the ubuntu "tray" (or maybe in panel). I have never had that, even when `emote` used to work.  

What I tried,
- use htop and find that emote *was* actually running (noticed it's a snap, I had forgotten).
- terminated it
- re-tried the shortcut
- tried to launch emote from the normal launcher
- tried to run emote from terminal
    - says "starting emote daemon"
    - no errors
- re-tried the shortcut, still unable to access the GUI
- of course, I could not undo my GUI change of the shortcut
- tried to create a custom shortcut for `emote` from the ubuntu keyboard / shortcut options (didn't work)
- (...)

What I searched  : 
- this github issues, readme, wiki
- multiple google queries, no one seems to have had same problem «can't launch emote», «after changed main shortcut»

What I did not try :
- log off / log in
- restart X

**Comment by stephan-dev** [comment]
For the record / for search engines, solution :  
One needs to focus on the double aspect of `emote` : the `daemon`, and the `GUI`.  
I had never seen this with another program, here's the thing with emote :  
If one lost access to emote, can't launch, and no emote icon in the tray,  

- launch emote from terminal
    - output : `daemon starting`
- in _**another terminal tab**_, launch emote _**a second time**_, with exactly the same command (sic!)

This runs the GUI. Done.  
Of course, now, one can undo any bad change made from the GUI menu (preferences... or `keyboard shortcuts`), since we recovered access to emote GUI.

**Comment by stephan-dev** [comment]
When ran from Terminal, the emote daemon's output clearly mention the GUI as the emote `second instance`.
Once this second instance is triggered _once_ from a second Terminal tab, my custom shortcut works to launch the GUI. But then, I get a known problem : (at least) 2 seconds delay before the GUI appears.
I tested reverting to default shortcut `ctrl-alt-e` and the delay disappears.

My theory is that the default shortcut seems hardcoded in the emote _daemon_ (so, no delay). But when the shorcut is changed from GUI, a second instance of emote has to be running (or been run recently once??) so that the shorcut does anything.

I reopened this issue because I thought it was fixed with me posting this workaround above, so that at least with a web search, no one gets stuck anymore. But if this could really be fixed (if confirmed as bug), it would of course be even better.

**Comment by tom-james-watson** [comment]
Yeah what you bring up in your last comment is actually now highlighted in the readme. I'm not sure why this happens, actually.
